### DISCUSSION TITLE
Create a "demo content / setup" to base all our documentation on and maybe create a demo.pods.io as "showcase"

### DISCUSSION THREAD

**Discussion by quasel** [pods-framework/pods#6634] [Q&A]
As discussed with @jimtrue it might be an good idea to create a default "demo" structure that includes all of our shiny stuff and could be used as base for podscasts, demos, tutorials, documentation ... might help and avoid reinventing the wheel :D. Any ideas what for a proper USE Case Scenario? 

Like Star Wars Topic from Josh or Books or ... 

An Future step could maybe even create it as "package"  (just setup no content) for users to grab and import and go from there ...

**Comment by quasel** [comment]
Maybe based on the dummy-content plugin from the they guys from Admin Columns Pro?

**Comment by quasel** [comment]
use the same structure for #4214 and #4215

**Comment by jimtrue** [comment]
I think this is a great idea, but we have a handful of things to consider:

- [ ] Is this Demo content for us to show what Pods can do, or is it content to test on or both?
- [ ] If it's a TEST site, we can't keep it in Multisite, because the Pods version would change depending on what section we're testing. We only run released Pods version on our live sites in Multisite and they share the same plugin.
- [ ] I like the idea of building configurations that will be used to show off/be used in our Demo's for links etc. Keeps us consistent, but it should be _multiple_ demos. Here's Custom Settings and how they can be used, Here's a taxonomy configuration with massive filters, etc. I'd say let's put together a list of them that show off the flexibility of our different fields and connections.
- [ ] You said 'demo content' for people to load: My only concern - we don't really clear out all the fields and meta when we delete fields. Until we have a cleaner 'install demo file' and 'clean up demo file' process, we can't offer this.

I'm opening this as a separate ticket on this one, but Deleting a Demo, a Pod, or it's fields, should Delete the Demo and it's data, without requiring people go into each field and 'Delete All' first and then Delete the field.
